torchvision/transforms/transforms.py

```diff
@@ -308,13 +308,27 @@ class Resize(torch.nn.Module):
             smaller edge may be shorter than ``size``. This is only supported
             if ``size`` is an int (or a sequence of length 1 in torchscript
             mode).
-        antialias (bool, optional): antialias flag. If ``img`` is PIL Image, the flag is ignored and anti-alias
-            is always used. If ``img`` is Tensor, the flag is False by default and can be set to True for
-            ``InterpolationMode.BILINEAR`` and ``InterpolationMode.BICUBIC`` modes.
-            This can help making the output for PIL images and tensors closer.
+        antialias (bool, optional): Whether to apply antialiasing.
```

Comment:
Updated the docstring hopefully making it more obvious. You're allowed to request changes but only once :) !
(I want to avoid having to update those in multiple places 10x in a row )

Reply:
I was wondering if we could later also opt to the similar idea as pytorch is doing for their docs: 

```python

geom_common_args = {
    "antialias": "<doctring>"
}

class ATranform(...)
"""Doc about ATrasform

    {antialias}
""".format(**geom_common_args)
```


https://github.com/pytorch/pytorch/blob/98e1b3e93a10b7dceeea7df489dafacbcaab556f/torch/_torch_docs.py#L190-L214

Reply:
So far we've found value in having the docstings easily available when reading the code. It's true that this kind of docstring engine becomes necessary when the same parameters are needed across many different APIs. We might have reached that point already, IDK. Happy to discuss more about this later